models/base_sources/aave_base_sources.yml

```diff
@@ -0,0 +1,40 @@
+version: 2

```

Comment:
this source file should be moved into the aave/ethereum directory along with the schema file and model itself. base sources is intended for base tables like ethereum transcations, traces, etc. 
decoded table sources should fit into their own file in their respective directory